tensorflow_serving/example/inception_client.cc

```diff
@@ -0,0 +1,179 @@
+#include <iostream>
+#include <fstream>
+
+#include <grpc++/create_channel.h>
+#include "tensorflow_serving/apis/prediction_service.grpc.pb.h"
+
+
+using grpc::Channel;
+using grpc::ClientContext;
+using grpc::ClientReader;
+using grpc::ClientReaderWriter;
+using grpc::ClientWriter;
+using grpc::Status;
+
+
+using tensorflow::serving::PredictRequest;
+using tensorflow::serving::PredictResponse;
+using tensorflow::serving::PredictionService;
+
+const std::string SERVING_HOST = "localhost:9000";
```

Comment:
Yes, please update SERVING_HOST to something like ADDRESS and make flag configurable.
Input file name should also be flag configurable.
And please s/SERVICE_NAME/MODEL_NAME, and make configurable. Can default to "inception".
Thanks!